gravitee-am-ui/src/app/domain/settings/password-policies/password-policies-idp-select-dialog/password-policies-idp-select-dialog.component.ts

```diff
@@ -69,17 +63,10 @@ export class PasswordPoliciesIdpSelectDialogComponent {
   }
 
   confirmSelection(): void {
-    this.dialogRef.close(this.result);
+    this.dialogRef.close(this.unlinkedResult);
   }
 
-  handleEvent(event: { id: string; selected: boolean }, array: string[]) {
-    if (event.selected) {
-      array.push(event.id);
-    } else {
-      const index = array.indexOf(event.id);
-      if (index !== -1) {
-        array.splice(index, 1);
-      }
-    }
+  handleEvent(event: { id: string; selected: boolean }): void {
+    this.unlinkedResult.set(event.id, event.selected);
   }
```

Comment:
The same thing. the dialog handles unlinkedIdps and linkedIdps. Here only one is handled? Or Im missing somthing?

Reply:
it will be handled here: [AM-2895](https://gravitee.atlassian.net/browse/AM-2895)